### PR TITLE
Pocket relayer tests

### PR DESCRIPTION
**Pull request by rem1niscence**
- [X] Minimal metrics recorder mock (for influxdb/pg (Postgres))
- [X] Small cleanup on cherry picker unit tests
- [X] fix Regex bug on pocket relayer when a non json request arrived at a enforced json blockchain
- [X] Pocket relayer tests

### CODE REVIEW

File: src/services/metrics-recorder.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import { HttpErrors } from '@loopback/rest'
 const logger = require('../services/logger')
 const os = require('os')
 
-const { InfluxDB, Point } = require('@influxdata/influxdb-client')
+import { InfluxDB, Point } from '@influxdata/influxdb-client'
```
Comment (Evalir) [inline src/services/metrics-recorder.ts:11]: Let's move this import upwards along with the other ESModule imports :).

File: tests/mocks/metricsRecorder.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { expect, sinon } from '@loopback/testlab'
```
Comment (Evalir) [inline tests/mocks/metricsRecorder.ts:1]: Let's remove the action warnings!

File: tests/mocks/metricsRecorder.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import { Pool } from 'pg'
+import rewiremock from 'rewiremock'
+import { InfluxDB, WriteApi } from '@influxdata/influxdb-client'
+import { MetricsRecorder } from '../../src/services/metrics-recorder'
+import { Redis } from 'ioredis'
```
Comment (Evalir) [inline tests/mocks/metricsRecorder.ts:6]: Let's move this import upwards so we only have relative paths below the `influx` import.

**Review by Evalir — APPROVED** [review]
LGTM

**Review by crisog — APPROVED** [review]

**Pull request merged**